src/types/redis_hash.cc

```diff
@@ -387,4 +388,58 @@ rocksdb::Status Hash::Scan(const Slice &user_key, const std::string &cursor, uin
   return SubKeyScanner::Scan(kRedisHash, user_key, cursor, limit, field_prefix, fields, values);
 }
 
+rocksdb::Status Hash::RandField(const Slice &user_key, std::vector<FieldValue> *field_values, int64_t command_count,
+                                bool noparmeter, HashFetchType type) {
+  uint64_t count = (command_count >= 0) ? static_cast<uint64_t>(command_count) : static_cast<uint64_t>(-command_count);
+  bool unique = (command_count >= 0);
+
+  std::string ns_key;
+  AppendNamespacePrefix(user_key, &ns_key);
+  HashMetadata metadata(/*generate_version=*/false);
+  rocksdb::Status s = GetMetadata(ns_key, &metadata);
+  if (!s.ok()) return s;
+
+  uint64_t size = metadata.size;
+  std::vector<FieldValue> samples;
+  // TODO: Getting all values in Hash might be heavy, consider lazy-loading these values later
+  if (count == 0 && !noparmeter) return rocksdb::Status::OK();
```

Comment:
Would we remove `noparameter` in `Hash::RandField` by default set `command_count_ == 1`?